src/main/groovy/cruise/umple/UmpleGradlePlugin.groovy

```diff
@@ -29,12 +29,14 @@ class UmpleGradlePlugin implements Plugin<Project> {
 		project.task('generateSource') << {		
 			// command line arguments can be specified through gradle by -P (-P is for project properties)
 			// eg: "gradle compileUmpleFile -PUMPLE_FILE_PATH=test.ump -PLANGUAGE_TO_GENERATE=Java"	
+			m_umpleFilePath = "${project.projectDir}${File.separator}" 
+			m_generatedOutputPath = "${project.projectDir}${File.separator}"
 		
 			if(project.hasProperty(UMPLE_FILE_PATH))
 			{
-				m_umpleFilePath = project.getProperty(UMPLE_FILE_PATH)	
+				m_umpleFilePath += project.getProperty(UMPLE_FILE_PATH)	
```

Comment:
This is direct changes of behaviour. Are the paths supposed to be relative to the project directory or can they be absolute? I would recommend using [`Path#relativize(Path)`](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Path.html#relativize(java.nio.file.Path)) to make sure the paths behave properly and consistently.